examine.lic

```diff
@@ -0,0 +1,25 @@
+class Examine
```

Comment:
💡 All scripts should have a link here to elanthipedia & an entry in elanthipedia with more info on the script and its args/etc.